app/Providers/NovaServiceProvider.php

```diff
@@ -37,7 +37,7 @@ protected function gate(): void
         Gate::define(
             "viewNova",
             function (BrewmapUser $user): bool {
-                return true;
+                return $user->isAdmin;
```

Comment:
Probably we could shorten it into `Gate::define("viewNova", fn(BrewmapUser $user): bool => $user->isAdmin)`